chapters/01-spatial-data.qmd

```diff
@@ -21,3 +29,118 @@ using GeoMakie
 
 f, a, p = poly(df.geom)
 ```
+
+
+### Raster from scratch {#sec-raster-from-scratch}
+
+In this section, we are going to demonstrate the creation of rasters from scratch.
+We will construct two small rasters, `elev` and `grain`, which we will use in examples later in the book.
+Unlike creating a vector layer (see @sec-vector-layer-from-scratch), creating a raster from scratch is rarely needed in practice because aligning a raster with the proper spatial extent is challenging to do programmatically ("georeferencing" tools in GIS software are a better fit for the job).
+Nevertheless, the examples will be helpful to become more familiar with the **Rasters.jl** data structures.
+
+```{julia}
+using Rasters
+import GeoFormatTypes as GFT
+```
+
+Conceptually, a raster is an array combined with georeferencing information, whereas the latter comprises:
+
+-   A transformation matrix, containing the origin and resolution, thus linking pixel indices with coordinates in a particular coordinate system
```

Comment:
A transformation matrix can also include shearing/rotation etc. Not sure if it should be discussed.

Reply:
Hmm, this was just copied from the Python definition.  Should we instead talk about ["lookups"](https://rafaqz.github.io/DimensionalData.jl/stable/api/lookuparrays#lookups)?